tools/openshift/ocp-ipi-aws/prep_for_subm.sh

```diff
@@ -1,82 +1,85 @@
-#!/bin/sh
+#!/bin/bash
 set -e
-OCP_INS_DIR="$(realpath ${1:-.})"
+
+# Get OCP installer path, or use current directory
+OCP_INS_DIR="$(realpath ${1:-.} 2>/dev/null || :)"
```

Comment:
This is still broken; how about

```suggestion
OCP_INS_DIR="$(realpath -- ${1:-.})"
if [ ! -d "${OCP_INS_DIR}" ]; then
  echo Please specify the OpenShift installation directory as the first argument, for example
  echo $0 ${PWD} $*
  exit 1
fi
```

Reply:
Can you show this broken output you get ?

Also, the problem with using realpath without `2>/dev/null || :`
that it can fail before that condition you've suggested. e,g:
```
OCP_INS_DIR="$(realpath -- "/no/dir")"
realpath: ‘/no/dir’: No such file or directory
```

Reply:
We could fix that by using `|| echo .` but, thinking about it a bit more, I think failing here is fine: the error message is clear, the user will know that they provided an invalid directory.

Reply:
Fixed as you suggested, thanks!